src/main/flight/mixer_tricopter.h

```diff
@@ -73,4 +73,82 @@ int16_t triGetMotorCorrection(uint8_t motorIndex);
  */
 _Bool triEnableServoUnarmed(void);
 
+#ifdef USE_SERVOS
```

Comment:
I understand, I think it is ok to do the cleaning later on as well. This should not be a blocker regarding the pull request.